apps/base-docs/docs/pages/identity/smart-wallet/features/sub-accounts.mdx

```diff
@@ -0,0 +1,9 @@
+# Sub Accounts
+
+Sub Accounts enable developers to create and manage app-specific accounts for their applications. These accounts are linked to the user's Smart Wallet and can be utilized to sign transactions specifically for the app. This integration grants developers full control over the user experience, ensuring a seamless and cohesive interaction within the app.
```

Comment:
This is pretty specific to the app version of sub accounts, but I think that's OK for now.

Reply:
Updated. Made it more sub account specific. Thanks for flagging!